package/src/main.rs

```diff
@@ -1083,75 +1080,103 @@ index b70ae75..271706a 100644
-        softlink(scie_pants_scie, &pants_from_sources)?;
-
-        test_pants_from_sources(
-            Command::new(pants_from_sources)
-                .env("SCIE_PANTS_TEST_MODE", "pants_from_sources mode")
-                .current_dir(user_repo_dir),
-            "The pants_from_sources mode is working.",
-        )?;
+        {
+            let side_by_side_root = create_tempdir()?;
+            let pants_dir = side_by_side_root.path().join("pants");
+            softlink(&pants_2_14_1_clone_dir, &pants_dir)?;
+            let user_repo_dir = side_by_side_root.path().join("user-repo");
+            ensure_directory(&user_repo_dir, true)?;
+            touch(user_repo_dir.join("pants.toml").as_path())?;
+            touch(user_repo_dir.join("BUILD_ROOT").as_path())?;
+
+            let pants_from_sources = side_by_side_root.path().join("pants_from_sources");
+            softlink(scie_pants_scie, &pants_from_sources)?;
+
+            test_pants_from_sources(
+                Command::new(pants_from_sources)
+                    .env("SCIE_PANTS_TEST_MODE", "pants_from_sources mode")
+                    .env("PANTS_VENV_DIR_PREFIX", &pants_2_14_1_venv_dir)
+                    .current_dir(user_repo_dir),
+                vec![
+                    "The pants_from_sources mode is working.",
+                    "Pants from sources argv: --no-verify-config -V.",
+                ],
+            )?;
+        }
 
         integration_test!("Verify delegating to `./pants`.");
-        let result = execute(
-            Command::new(scie_pants_scie)
-                .arg("-V")
-                .env("SCIE_PANTS_TEST_MODE", "delegate_bootstrap mode")
-                .current_dir(pants_2_14_1_clone_dir)
-                .stderr(Stdio::piped()),
-        )?;
-        let stderr = String::from_utf8(result.stderr).map_err(|e| {
-            Code::FAILURE.with_message(format!("Failed to decode Pants stderr: {e}"))
-        })?;
-        let expected_message = "The delegate_bootstrap mode is working.";
-        assert!(
-            stderr.contains(expected_message),
-            "STDERR did not contain '{expected_message}':\n{stderr}"
-        );
-        let expected_argv_message = "Pants from sources argv: -V.";
-        assert!(
-            stderr.contains(expected_argv_message),
-            "STDERR did not contain '{expected_argv_message}':\n{stderr}"
-        );
+        {
+            test_pants_from_sources(
+                Command::new(scie_pants_scie)
+                    .env("SCIE_PANTS_TEST_MODE", "delegate_bootstrap mode")
+                    .current_dir(&pants_2_14_1_clone_dir),
+                vec![
+                    "The delegate_bootstrap mode is working.",
+                    "Pants from sources argv: -V.",
+                ],
+            )?;
+        }
+
+        let pants_release = "2.16.0.dev5";
+        integration_test!("Verify usage of a Pants {pants_release} on the pants repo.");
```

Comment:
Will fix on next push:

```suggestion
        integration_test!("Verify usage of Pants {pants_release} on the pants repo.");
```